WebUI/static/css/page404.css

```diff
@@ -0,0 +1,19 @@
+.center404{
```

Comment:
Isn't there a site.css that this can all be part of / derived from? Generally we want to try and re-use styles if we can.